--- src/raspberrypi/devices/cfilesystem.cpp	2022-09-08 12:07:14.000000000 +0100
+++ src/raspberrypi/devices/cfilesystem.cpp.patched	2022-09-08 12:12:55.000000000 +0100
@@ -1075,12 +1075,15 @@
 			m_dirHuman.name[i] = ' ';
 	}
 
-	for (i = 0; i < 10; i++) {
-		if (p < m_pszHumanExt)
-			m_dirHuman.add[i] = *p++;
-		else
-			m_dirHuman.add[i] = '\0';
-	}
+	// This code causes a compilation error on Debian "bullseye" (amd64)
+	// https://github.com/akuker/RASCSI/issues/821
+	// 
+	// for (i = 0; i < 10; i++) {
+	// 	if (p < m_pszHumanExt)
+	// 		m_dirHuman.add[i] = *p++;
+	// 	else
+	// 		m_dirHuman.add[i] = '\0';
+	// }
 
 	if (*p == '.')
 		p++;
